opentelemetry-api/opentelemetry/internal/__init__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2019, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from .version import __version__
```

Comment:
In OpenCensus we avoid the relative import. I wonder how people think about using relative versus absolute import in OpenTelemetry.

The following article explored the pros and cons.
https://realpython.com/absolute-vs-relative-python-imports/

Reply:
I like relative imports and use them in personal projects. The [google style guide](http://google.github.io/styleguide/pyguide.html#23-packages) says to import modules using the full path, but the justification for doing this only applies to _implicit_ relative imports:

> Avoids conflicts in module names or incorrect imports due to the module search path not being what the author expected

Since implicit relative imports are [gone in python 3](https://docs.python.org/3.0/whatsnew/3.0.html#removed-syntax) I think this is fine. If we decide to use it we should make sure we use this import style everywhere.

Reply:
@carlosalberto @Oberon00 I'll merge the PR first, please comment here if you see potential issue that could bite us. Thanks.